andela/wg-logros
# #155322966 Fix social login bug

#### What does this PR do?
Fix a bug that is causing the social login i.e with Twitter, Facebook, and Google not to work.
#### Description of Task to be completed?
The current develop branch settings module has two `AUTHENTICATION_BACKENDS`. For the social login,s to work, we need to remove one.
#### How should this be manually tested?
On the login page, when one tries to login with either Twitter, Facebook or Google, the application should be able to log the user in.
#### What are the relevant pivotal tracker stories?
[155322966](https://www.pivotaltracker.com/story/show/155322966)
#### Screenshots (if appropriate)
* With the bug in place
<img width="1031" alt="screen shot 2018-02-19 at 12 41 37" src="https://user-images.githubusercontent.com/15800486/36371163-5092c50e-1572-11e8-9117-cebbc50681a1.png">
* With the bug fixed
<img width="1160" alt="screen shot 2018-02-19 at 12 42 52" src="https://user-images.githubusercontent.com/15800486/36371200-716445aa-1572-11e8-91f3-79df269e30e9.png">
